hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -362,6 +363,15 @@ StartContainerBalancerResponseProto startContainerBalancer(
    */
   List<String> getScmRatisRoles() throws IOException;
 
+  /**
+   * Force generates new secret keys (rotate).
+   *
+   * @param force boolean flag that forcefully rotates the key on demand
+   * @return
+   * @throws TimeoutException
+   */
+  boolean checkAndRotate(boolean force) throws TimeoutException, IOException;
```

Comment:
nit: maybe name it `rotateSecretKeys`. Note that this class name has no SecretKey context, so the method name needs it.